org.siril.Siril.json

```diff
@@ -18,36 +18,227 @@
+                "-DWITH_QT=OFF",
+                "-DWITH_TBB=OFF",
+                "-DWITH_XINE=OFF",
+                "-DBUILD_JPEG=ON",
+                "-DBUILD_TIFF=ON",
+                "-DBUILD_PNG=ON"
             ],
             "sources": [
                 {
                     "type": "archive",
+                    "url": "https://github.com/opencv/opencv/archive/refs/tags/4.12.0.zip",
+                    "sha256": "fa3faf7581f1fa943c9e670cf57dd6ba1c5b4178f363a188a2c8bff1eb28b7e4"
+                }
+            ]
+        },
+        {
+            "name": "cfitsio",
+            "config-opts": [
+                "--enable-reentrant",
+                "CFLAGS=-std=c99"
+            ],
+            "make-args": ["shared"],
+            "cleanup": [
+                "/include",
+                "*.a",
+                "*.la",
+                "/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.2.0.tar.gz",
                     "disable-http-decompression": true,
-                    "url": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.4.0.tar.gz",
-                    "sha256": "95900cf95ae760839e7cb9678a7b2fad0858d6ac12234f934bd1cb6bfc246ba9"
+                    "sha256": "eba53d1b3f6e345632bb09a7b752ec7ced3d63ec5153a848380f3880c5d61889"
                 }
             ]
         },
+        {
+            "name": "libxml2",
+            "buildsystem": "autotools",
+            "config-opts": [
+                "--disable-static",
+                "--disable-python",
+                "--disable-introspection",
+                "--with-history",
+                "--with-icu"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libxml2/2.11/libxml2-2.11.5.tar.xz",
+                    "sha256": "3727b078c360ec69fa869de14bd6f75d7ee8d36987b071e6928d4720a28df3a6"
+                }
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/cmake",
+                "/bin/xmlcatalog",
+                "/bin/xmllint",
+                "/share/aclocal",
+                "/share/doc",
+                "/share/gtk-doc",
+                "/share/man"
+            ]
+        },
         {
             "name": "gtksourceview4",
             "buildsystem": "meson",
+            "config-opts": [
+                "-Dgir=false",
+                "-Dvapi=false"
+            ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gtksourceview/4.8/gtksourceview-4.8.4.tar.xz",
+                    "url": "https://deb.debian.org/debian/pool/main/g/gtksourceview4/gtksourceview4_4.8.4.orig.tar.xz",
```

Comment:
what's wrong with using GNOME.org. Debian archive get purged so this will break.

Reply:
Ok, reverted